cloud/README.md

```diff
@@ -1,11 +1,26 @@
 # Cloud Technical Assessment
 
-### Key Concepts
+## Key Concepts
 
-- AWS
-- Docker Deployments
-- Architectural Understanding
+- Able to implement a viable CI-CD that automate deployments
+- Able to design a scalable cloud architecture
+- Able to implement the designed architecture into a cloud platform of your choose
+- Able to justify the design decisions made, including budget, security, performance and scalability
 
 ## Resources
 
+This is just a list of resources that were found relevant. You don't have to go through all of them.
+
+- [What is DevOps? REALLY understand it | DevOps vs SRE](https://www.youtube.com/watch?v=0yWAtQ6wYNM)
+- [What is GitOps, How GitOps works and Why it's so useful](https://www.youtube.com/watch?v=f5EpcWp0THw)
+- [DevOps CI/CD Explained in 100 Seconds](https://www.youtube.com/watch?v=scEDHsr3APg)
+- [Containers on AWS Overview: ECS | EKS | Fargate | ECR](https://www.youtube.com/watch?v=AYAh6YDXuho&pp=ygUPYXdzIGVjcyBmYXJnYXRl)
+- [What is Infrastructure as Code? Difference of Infrastructure as Code Tools](https://www.youtube.com/watch?v=POPP2WTJ8es)
+- [Terraform Tutorial for Beginners + Labs: Complete Step by Step Guide!](https://www.youtube.com/watch?v=YcJ9IeukJL8&list=WL&index=44)
+
 ## Tasks
+
+1. Set up a CI-CD pipeline that automates the deployment of the application
```

Comment:
I assume by this, you mean they have to deploy all the services defined [here](https://github.com/UNSW-Redback-Racing/DAQ-Technical-Assessment/tree/2024-rewrite/telemetry)?

Reply:
Should we also give them hints/tips?

Reply:
added a preface for the tasks.

Reply:
> I assume by this, you mean they have to deploy all the services defined [here](https://github.com/UNSW-Redback-Racing/DAQ-Technical-Assessment/tree/2024-rewrite/telemetry)?

^

Reply:
If this is true, should I change the docker compose to use images pulled from docker hub instead. Because at the moment, all the services within `telemetry/` are containerised with Dockerfiles.

Reply:
They can follow what you have set up, but they should have the freedom to rewrite everything. if they want to do so they know what they are doing.